pkgpanda/__init__.py

```diff
@@ -463,14 +475,23 @@ def add_user(self, username):
                                   "automatic user addition is disabled".format(username))
 
         # Add the user:
+        add_user_cmd = [
```

Comment:
Along the same thing as the "if we aren't allowed to add users, error and exit" just above this, we need to do that same check for groups. If add_user is called, and the group already exists, and the user belongs to the group, no-op. Otherwise print an error message that things aren't pre-configured properly.